README.md

```diff
@@ -53,14 +45,14 @@ Note: Nextjs uses `.env.development` and `.env.production` for their respective
 
 1. `cp .env.example .env.local` and fill out variables
 2. `npm install`
-3. `npm run dev` and point browser to http://localhost:3000/library-card/new
+3. `npm run dev` and point browser to http://localhost:3001/library-card/new
```

Comment:
`Dockerfile` and `docker-compose.yml` still have the app running on port 3000 locally, do we need to update those as well?

https://github.com/NYPL/nypl-library-card-app/blob/main/Dockerfile#L13
https://github.com/NYPL/nypl-library-card-app/blob/main/docker-compose.yml#L12

Reply:
Agree with Russell, we should keep the port at 3000 for `npm run dev`, that's what nextjs uses by default.